docs/sphinx/sections/api/apidocs/libraries/dagster-pagerduty.rst

```diff
@@ -30,5 +30,5 @@ alerts from within your ops.
 
 .. currentmodule:: dagster_pagerduty
 
-.. autoconfigurable:: pagerduty_resource
```

Comment:
should we keep the legacy api maybe in a "Legacy" section?

Reply:
good idea 👍